plugins/woo-ai/src/utils/productData.ts

```diff
@@ -102,8 +102,16 @@ const getProductType = () => {
 	return productTypeEl ? productTypeEl.value : '';
 };
 
+const getProductId = (): number => {
```

Comment:
Just realized this is redudant with [this little util](https://github.com/woocommerce/woocommerce/blob/b6fed99d4de602f1c9646e8d1719d381ef02391e/plugins/woo-ai/src/utils/get-post-id.ts#L1). They're used a little differently, but it would make sense to consolidate them in some way.